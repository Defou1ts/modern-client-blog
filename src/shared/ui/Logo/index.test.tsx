import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { Logo } from './index';

describe('Logo', () => {
	it('renders Logo', () => {
		render(<Logo />);
		const text = screen.getByText('Modsen Client Blog');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('Logo  snapshot', () => {
		const component = renderer.create(<Logo />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
