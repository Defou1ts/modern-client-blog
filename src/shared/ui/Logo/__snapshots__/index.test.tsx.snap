// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo  snapshot 1`] = `
<a
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  <h1
    className="logo"
  >
    Modsen Client Blog
  </h1>
</a>
`;
